modules/statistics/RTPStatsCollector.js

```diff
@@ -401,6 +404,27 @@ StatsCollector.prototype._defineGetStatValueMethod = function(keys) {
     return (item, name) => itemStatByKey(item, keyFromName(name));
 };
 
+/**
+ * Obtains a stat value from given stat and converts it to a non-negative
+ * number. If the value is either invalid or negative then 0 will be returned.
+ * @param report
+ * @param {string} name
+ * @return {number}
+ */
+StatsCollector.prototype.getNonNegativeStat = function(report, name) {
+    let value = this._getStatValue(report, name);
```

Comment:
That number is larger than 2147483647 (2^32 -1) so the code is actually working even better, since that's not a valid SSRC number, isn't it?